robottelo/hosts.py

```diff
@@ -71,6 +71,18 @@
 }
 
 
+@lru_cache(maxsize=128)
+def lru_sat_ready_rhel(rhel_ver):
+    deploy_args = {
+        'deploy_rhel_version': rhel_ver or settings.server.version.rhel_version,
+        'deploy_flavor': 'satqe-ssd.standard.std',
+        'promtail_config_template_file': 'config_sat.j2',
+        'workflow': 'deploy-rhel',
```

Comment:
Did you intend to leave these as-is, or were you planning on moving these to config?

Reply:
Leaving it as it is!